apps/quorom/src/api.rs

```diff
@@ -0,0 +1,145 @@
+use std::{net::SocketAddr, time::Duration};
+use axum::{routing::{get, post}, Json, Router};
+use secp256k1::{ecdsa::{RecoverableSignature, RecoveryId}, Message, Secp256k1};
+use serde::{Deserialize};
+use sha3::{Digest, Keccak256};
+use ::time::OffsetDateTime;
+use wormhole_sdk::{vaa::{Body, Header, Signature}, GuardianAddress, GuardianSetInfo, Vaa};
+
+use crate::{server::State, ws::{ws_route_handler, UpdateEvent}};
+
+const OBSERVATION_LIFETIME: u32 = 10; // In seconds
```

Comment:
Can we make this configurable? In any case there's a delay in the network i want to be able to tune it a bit.